python/srv

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 
-# Copyright 2017 CodiLime
+# Copyright 2016-2017 CodiLime
```

Comment:
Up - do we want to have copyright notices for whole duration of our project on all files - not starting when file was introduced?